doc/maintenance.md

```diff
@@ -167,7 +167,7 @@ alongside the rest of the Scientific Python ecosystem, and therefore supports:
 * All minor versions of NumPy released in the 24 months prior to the project,
   and at minimum the last three minor versions.
 
-In `setup.py`, the `python_requires` variable should be set to the minimum
+In `pyproject.toml`, the `requires-python` key should be set to the minimum
```

Comment:
We have the following command at line 280:
```
python setup.py --long-description | rst2html.py --no-raw > index.html
```
After removing `setup.py`, the command is no longer valid.

Reply:
I've replaced the command with:
```
rst2html.py --no-raw README.rst > index.html
```
in 7553f0b4.